src/thunderbots/software/test/ai/hl/stp/evaluation/pass.cpp

```diff
@@ -0,0 +1,187 @@
+/**
```

Comment:
Rename this file to `intercept` as well to match the implementation file.

Reply:
done